src/app.hooks.js

```diff
@@ -5,10 +5,11 @@ const sanitize = (data, options) => {
 	// https://www.npmjs.com/package/sanitize-html
 	if ((options||{}).html === true) {
 		// editor-content data
+		// TODO what is 'rechnen' used for?
```

Comment:
This should not stay in the code. @schurigh: [you added](https://github.com/schul-cloud/schulcloud-server/commit/8f000812c53e7741136653208366753c8a9b10d7#diff-fd78da2a511feb8d56db9a14111540f5R10) `rechnen`, what is it used for?

Reply:
hm, not sure, couldn't find any element that needs this...